pkg/deps/maven_test.go

```diff
@@ -106,14 +132,22 @@ func TestResolveMaven(t *testing.T) {
 				<artifactId>skywalking-sharing-server-plugin</artifactId>
 				<version>8.6.0</version>
 			</dependency>
+			<dependency>
+				<groupId>com.fasterxml.jackson.datatype</groupId>
+				<artifactId>jackson-datatype-jsr310</artifactId>
+				<version>2.13.3</version>
+			</dependency>
```

Comment:
> I can't follow the PR description. What does this change mean?

Previously when the tool failed to identify a dependency's license, we have to create a demo maven project and run the tool in debug mode, now we can just put the dependency in the test codes and debug the test, like this.